app/models/miq_queue_worker_base/runner.rb

```diff
@@ -64,6 +64,7 @@ def get_message_via_drb
         _log.debug("#{log_prefix} #{MiqQueue.format_short_log_msg(msg)} stale, retrying...")
         next
       rescue => err
+        msg.update_column(:state, MiqQueue::STATUS_ERROR)
```

Comment:
What if this fails?  Say, if the database is down?  should we do some kind of `rescue nil` here?